.github/workflows/black.yaml

```diff
@@ -10,4 +10,4 @@ jobs:
       - uses: psf/black@stable
         with:
           version: "23.12.1"
-          args: ". --check --extend-exclude '.*_version\.py$'"
+          options: ". --check --extend-exclude '.*_version\\.py$'"
```

Comment:
Could you the `--verbose` flag so that we see what exactly is being tested? Thanks!